metatron-app/metatron-discovery
# F#834 local file snapshot generating error

### Description
Fix local file snapshot generating error.
Add avoid logic null point exception error by fileURI parameter.

**Related Issue** : 
https://github.com/metatron-app/metatron-discovery/issues/834


### How Has This Been Tested?
1. go to data preparation menu
2. select data flow and data set
3. click snapshot button
4. select local and csv
5. check generated snapshot and download it.


### Types of changes



### Checklist:



### Additional Context
